src/Billable.php

```diff
@@ -594,9 +605,15 @@ public function updateDefaultPaymentMethodFromStripe()
      */
     protected function fillPaymentMethodDetails($paymentMethod)
     {
-        if ($paymentMethod->type === 'card') {
-            $this->card_brand = $paymentMethod->card->brand;
-            $this->card_last_four = $paymentMethod->card->last4;
+        switch ($paymentMethod->type) {
+            case 'card':
+                $this->card_brand = $paymentMethod->card->brand;
+                $this->card_last_four = $paymentMethod->card->last4;
+                break;
+            case 'sepa_debit':
+                $this->card_brand = 'SEPA Debit';
```

Comment:
following the example of `Bank Account` here .. obviously we could also store the `bank_code` here but I guess in most cases people just want to know if its a SEPA payment method.